### PR TITLE
Add script for fixing descriptionUpgrader broken by a previous script

### PR DESCRIPTION
**Pull request by andersju**

### CODE REVIEW

**Comment by niklasl** [comment]
Nice! We *may* also want to check if the latest descriptionUpgrader list has had any new items added since the old script was run? (Something like fixedList += newestList - brokenResultList...) Not sure about the likelihood of that though.

**Review by olovy — APPROVED** [review]
LGTM! :ship:

**Comment by olovy** [comment]
> Nice! We _may_ also want to check if the latest descriptionUpgrader list has had any new items added since the old script was run? (Something like fixedList += newestList - brokenResultList...) Not sure about the likelihood of that though.

Or just step through the rest of the versions and check that it hasn't changed?

**Comment by andersju** [comment]
> Nice! We _may_ also want to check if the latest descriptionUpgrader list has had any new items added since the old script was run? (Something like fixedList += newestList - brokenResultList...) Not sure about the likelihood of that though.

Ah, you mentioned that before but it slipped my mind. Should be fixed now!

File: whelktool/scripts/cleanups/2020/12/lxl-3457-fix-descriptionUpgrader.groovy [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            }
+        }
+    }
+
+    if (changed) {
+        // Check whether something has been added to descriptionUpgrader since it was broken
+        if ((currentBrokenList - listWhenItBroke).size() > 0) {
+            fixedList += currentBrokenList - listWhenItBroke
+            hasAdditions.println("${data.doc.getURI()}")
+        }
```
Comment (olovy) [inline whelktool/scripts/cleanups/2020/12/lxl-3457-fix-descriptionUpgrader.groovy:55]: :star:

**Review by olovy — APPROVED** [review]
:ship:

**Pull request merged**